front/app/hooks/useNavbarItems.ts

```diff
@@ -9,20 +9,20 @@ import { truncateMultiloc } from 'utils/textUtils';
 
 export type TNavbarItemsState = INavbarItem[] | undefined | null | Error;
 
-export default function useNavbarItems() {
+export default function useNavbarItems({ standard } = { standard: false }) {
```

Comment:
What is standard? Does it refer to the price plan? If so - I would suggest to pass the price plan explicitly and properly typed and then check if it's standard in the component itself. These boolean properties can easily get confusing and hard to reason about.